pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -394,54 +394,50 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	Context("with VirtualMachineInstance metadata", func() {
 		DescribeTable(
 			"Should allow VMI creation with kubevirt.io/ labels only for kubevirt service accounts",
-			func(vmiLabels map[string]string, userAccount string, positive bool) {
-				vmi := api.NewMinimalVMI("testvmi")
-				vmi.Labels = vmiLabels
+			func(labels map[string]string, userAccount string) {
+				vmi := newBaseVmi()
+				vmi.ObjectMeta.SetLabels(labels)
```

Comment:
nit: AFAIK, There is no single standard in the project, but this could also be achieved by:
```suggestion
				vmi.ObjectMeta.Labels = labels
```
Same for annotations.